README.md

```diff
@@ -98,3 +98,4 @@ Pro tip: watch this repository to get notified about new plugins.
 - [tmux-yank](https://github.com/tmux-plugins/tmux-yank) - Plugin for copying
   to system clipboard. Works on MacOS, Linux and Cygwin.
 - [tpm](https://github.com/tmux-plugins/tpm) - Tmux plugin manager.
+- [tmux-ip-address](https://github.com/anghootys/tmux-ip-address) - Plugin for show public IP on status bar of tmux.
```

Comment:
Can you please move this line so all entries in this file stay alphabetical? Thanks

Reply:
Yes, Sure. I am doing this right now.